pkg/config/config_network.yaml

```diff
@@ -1,14 +1,13 @@
 testnet:
-  DNS: AS12YMz7NjyP3aeEWcSsiC58Hba8UxHapfGv7i4PmNMS2eKfmaqqC
+  DNS: AS12MGDGVB3xNDqzGidJM5fcAbjF1ZP5XAecm5ww4RU7sfQd4cMKk
   URLs:
     - https://test.massa.net/api/v2
+  Default: true
```

Comment:
are we sure that we want to keep testnet as default network ?

Reply:
no, true I just changed it, I was wainting for the buildnet to pass to v24